test.js

```diff
@@ -0,0 +1,26 @@
+var app = require('./app.js').app;
+var request = require('supertest')(app);
+
+describe('loading express', function() {
+	it('GET /', function testSlash(done) {
```

Comment:
Would you mind renaming the `testSlack` to `testRoot`. Helps make it a bit more clear :D